notation.go

```diff
@@ -203,6 +206,18 @@ func Verify(ctx context.Context, verifier Verifier, repo registry.Repository, re
 
 			// verify each signature
 			outcome, err := verifier.Verify(ctx, artifactDescriptor, sigBlob, opts)
+			// process verification items
+			for _, result := range outcome.VerificationResults {
+				if result.Error == nil {
+					continue
+				}
+				switch result.Action {
+				case trustpolicy.ActionLog:
+					logger.Warnf("verification failed on %v validation for signature %v but set to pass by verification action \"logged\". Reason: %v", result.Type, sigManifestDesc.Digest, result.Error)
+				case trustpolicy.ActionEnforce:
+					logger.Errorf("verification failed on %v validation for signature %v. Reason: %v", result.Type, sigManifestDesc.Digest, result.Error)
```

Comment:
will Error be logged when --debug is enabled or it will be always logged?

Reply:
According to current design, warning & error level log will always be shown. We should reach to a consensus.
@shizhMSFT @yizha1

Reply:
@priteshbandi The idea is there are 4 log levels: `error`, `warning`, `info` and `debug`. By default (without `--debug` flag), only `error` and `warning` logs are printed out, with `--debug` enabled, besides `error` and `warning`, `info` and `debug` level  logs will be printed out. In the future, we could introduce `--verbose` flag to print out `error`, `warning` and `info` logs. Does it make sense?

Reply:
Lets talk about this.